src/js/node/fs.ts

```diff
@@ -309,7 +309,13 @@ var access = function access(...args) {
   appendFileSync = fs.appendFileSync.bind(fs),
   closeSync = fs.closeSync.bind(fs),
   copyFileSync = fs.copyFileSync.bind(fs),
-  existsSync = fs.existsSync.bind(fs),
+  existsSync = function existsSync() {
```

Comment:
Don't forget about `fs.exists`, the synchronous callback-calling kind

https://github.com/nodejs/node/blob/c82f3c9e80f0eeec4ae5b7aedd1183127abda4ad/lib/fs.js#L248-L266

Reply:
that one's already try/catch'd but I'll add a test

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/fs.d.ts#L3468 maybe I shouldn't?